CIP-0021/README.md

```diff
@@ -5,6 +5,7 @@ Authors: Gabriel Kerekes <gabriel.kerekes@vacuumlabs.com>, Rafael Korbas <rafael
 Status: Draft
 Type: Standards
 Created: 2021-06-15
+Updated: 2021-11-23 multi-sig
```

Comment:
No need for this, we use `git` already :+1:

Reply:
Makes sense 👍  Removed https://github.com/cardano-foundation/CIPs/pull/157/commits/394645169f4b144044ff1289b48d95db9e8306f7.